com.unity.formats.fbx/Editor/ConvertToNestedPrefab.cs

```diff
@@ -930,27 +930,16 @@ internal static void CopyComponents(GameObject to, GameObject from, GameObject r
                 }
 
                 FixSceneReferences(fromComponent, toComponent, root);
-
-                // Do not try to copy materials for ParticleSystemRenderer, since it is not in the
-                // FBX file
-                if (fromComponent is Renderer && !(fromComponent is ParticleSystemRenderer))
-                {
-                    var renderer = toComponent as Renderer;
-                    var sharedMaterials = renderer.sharedMaterials;
-                    EditorJsonUtility.FromJsonOverwrite(json, toComponent);
-                    renderer.sharedMaterials = sharedMaterials;
-                }
+                
                 // SkinnedMeshRenderer stores both the mesh and materials.
                 // Make sure these do not get copied over when the SkinnedMeshRenderer is updated.
-                else if (fromComponent is SkinnedMeshRenderer)
+                if (fromComponent is SkinnedMeshRenderer)
                 {
                     var skinnedMesh = toComponent as SkinnedMeshRenderer;
                     var mesh = skinnedMesh.sharedMesh;
-                    var materials = skinnedMesh.sharedMaterials;
                     EditorJsonUtility.FromJsonOverwrite(json, toComponent);
                     var toSkinnedMesh = toComponent as SkinnedMeshRenderer;
```

Comment:
I think skinnedMesh and toSkinnedMesh are the same; while you're here, let's simplify this code a tiny bit